testsuite/ext-tools/maintenance_json_generator.py

```diff
@@ -0,0 +1,201 @@
+    "sle15_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                     "/SUSE_Updates_SLE-Product-SLES_15-LTSS_x86_64/"],
+    "sle15_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                     "/SUSE_Updates_SLE-Product-SLES_15-LTSS_x86_64/"],
+    "sle15sp1_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"],
+    "sle15sp1_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"],
+    "sle15sp2_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Product-SLES_15-SP2-LTSS_x86_64/"],
+    "sle15sp2_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Product-SLES_15-SP2-LTSS_x86_64/"],
+    "sle15sp3_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Basesystem_15-SP3_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Server-Applications_15-SP3_x86_64/"],
+    "sle15sp3_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Basesystem_15-SP3_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Server-Applications_15-SP3_x86_64/"],
+    "sle15sp4_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Basesystem_15-SP4_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Server-Applications_15-SP4_x86_64/"],
+    "sle15sp4_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Basesystem_15-SP4_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Server-Applications_15-SP4_x86_64/"],
+    "ceos7_client": "/SUSE_Updates_RES_7-CLIENT-TOOLS_x86_64/",
+    "ceos7_minion": "/SUSE_Updates_RES_7-CLIENT-TOOLS_x86_64",
+    "rocky8_minion": "/SUSE_Updates_RES_8-CLIENT-TOOLS_x86_64/",
+    "ubuntu1804_minion": "/SUSE_Updates_Ubuntu_18.04-CLIENT-TOOLS_x86_64/",
+    "ubuntu2004_minion": "/SUSE_Updates_Ubuntu_20.04-CLIENT-TOOLS_x86_64/",
+    "ubuntu2204_minion": "/SUSE_Updates_Ubuntu_22.04-CLIENT-TOOLS_x86_64/",
+    "debian9_minion": "/SUSE_Updates_Debian_9.0-CLIENT-TOOLS_x86_64/",
+    "debian10_minion": "/SUSE_Updates_Debian_10-CLIENT-TOOLS_x86_64/",
+    "debian11_minion": "/SUSE_Updates_Debian_11-CLIENT-TOOLS_x86_64/",
+    "opensuse153arm_minion": "/SUSE_Updates_openSUSE-SLE_15.3/",
+    "opensuse154arm_minion": "/SUSE_Updates_openSUSE-SLE_15.4/",
+    "rhel9_minion": "/SUSE_Updates_EL_9-CLIENT-TOOLS_x86_64/",
+    "rocky9_minion": "/SUSE_Updates_EL_9-CLIENT-TOOLS_x86_64/",
+    "alma9_minion": "/SUSE_Updates_EL_9-CLIENT-TOOLS_x86_64/",
+    "oracle9_minion": "/SUSE_Updates_EL_9-CLIENT-TOOLS_x86_64/",
+    "slemicro52_minion": ["/SUSE_Updates_SLE-Manager-Tools-For-Micro_5_x86_64/",
```

Comment:
Seems 5.3 is out, so you are missing it?

Reply:
Yes it is missing adding now